internal/support/notifications/errors/types.go

```diff
@@ -39,3 +39,31 @@ func (e ErrSubscriptionNotFound) Error() string {
 func NewErrSubscriptionNotFound(slug string) error {
 	return ErrSubscriptionNotFound{slug: slug}
 }
+
+type ErrMailAddrContainsCRLF struct {
```

Comment:
Great call on making a specific error type for these types of errors. I like this approach. However, can we make this error type a little more general, so we can also use this for other email related errors we might encounter, like invalid format. We can possibly rename this to `ErrInvalidEmailAddresses`? If we do make this more general we can probably rename `slug` to `description` and in the case of the `CRLF` error we can state that the email contains those invalid characters.

Reply:
yes, done.